installer/unix/install-cli

```diff
@@ -1,26 +1,23 @@
 #!/bin/bash
 # Install the Particle Command Line Interface (CLI)
 #
-# This installs a binary wrapper to the home directory.
+# This installs a binary packaged with Node and all the JS files to the home directory.
 #
-# That binary wrapper will download its own version of Node to ~/.particle
+# When a new version of the particle-cli is released, the binary will update to the latest version
+# in the background.
 #
-# It will also download the particle-cli npm module which contains the
-# actual Particle CLI logic to ~/.particle/node_modules.
-# When a new version of the particle-cli is released, the binary wrapper
-# will update to the latest version in the background.
-#
-# Run VERSION=3.2.0 ./install-cli to use the beta version
+# Run VERSION=3.2.0 ./install-cli to use the specific version
 
 VERSION=${VERSION}
 BINARY="particle"
 DEST_PATH="$HOME/bin"
 DEST="$DEST_PATH/$BINARY"
-MANIFEST_URL="https://binaries.particle.io/particle-cli/manifest.json"
+MANIFEST_HOST=${MANIFEST_HOST:-binaries.particle.io}
```

Comment:
Use `MANIFEST_HOST=binaries.staging.particle.io VERSION=3.23.0 ./install-cli` for testing